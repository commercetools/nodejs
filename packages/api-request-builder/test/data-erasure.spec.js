import withFullDataErasure from '../src/data-erasure'

describe('withFullDataErasure', () => {
  let service

  beforeEach(() => {
    service = { params: {}, withFullDataErasure }
  })

  test('should set the dataErasure option to true', () => {
    service.withFullDataErasure()
    expect(service.params.dataErasure).toMatchSnapshot()
  })
})
