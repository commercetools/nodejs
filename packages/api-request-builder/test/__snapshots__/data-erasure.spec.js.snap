// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withFullDataErasure should set the dataErasure option to true 1`] = `"dataErasure=true"`;
